include/zephyr/logging/log_core.h

```diff
@@ -509,7 +509,7 @@ void z_log_printf_arg_checker(const char *fmt, ...)
  * @param fmt            String to format.
  * @param ap             Pointer to arguments list.
  */
-static inline void log2_generic(uint8_t level, const char *fmt, va_list ap)
+static inline void log_generic(uint8_t level, const char *fmt, va_list ap)
```

Comment:
you want t ochange the @brief and remove reference to "logging v2"

Reply:
Fixed!